photon-core/src/main/java/org/photonvision/common/hardware/metrics/MetricsManager.java

```diff
@@ -245,8 +245,10 @@ public double getUptime() {
     public void publishMetrics() {
         logger.debug("Publishing Metrics...");
 
+        String[] topicParts = metricPublisher.getTopic().getName().split("/");
```

Comment:
Iirc there's a named constant in ntcore for / if you care. 

https://github.wpilib.org/allwpilib/docs/release/java/src-html/edu/wpi/first/networktables/NetworkTable.html#line-23